ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,898 @@
+
+Overview
+--------
+
+This tuturial will show you how to use the `Confluent Cloud CLI
+<https://docs.confluent.io/current/cloud/cli/install.html#ccloud-install-cli?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+to interact with your `Confluent Cloud
+<https://confluent.cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+cluster.
+
+
+Prerequisites
+~~~~~~~~~~~~~~
+
+-  Access to a |ccloud| cluster
+-  Local install  `Confluent Cloud
+   CLI <https://docs.cofonfluent.io/current/cloud/cli/install.html#ccloud-install-cli?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+   v1.7.0 or later
+-  |ccloud| user credentials saved in ``~/.netrc`` (save with command ``ccloud login --save``)
+-  Docker and Docker Compose for the local |kconnect| worker
+-  ``timeout`` installed on your host
+-  ``mvn`` installed on your host
+-  ``jq`` installed on your host
+
+Confluent Cloud Promo Code
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first 20 users to sign up for `Confluent Cloud
+<https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+and use promo code ``C50INTEG`` will receive an additional $50 free usage
+(`details
+<https://www.confluent.io/confluent-cloud-promo-disclaimer/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__).
+
+
+Run the demo
+------------
+
+`start.sh <start.sh>`__ is a fully scripted demo that shows you how to interact
+with |ccloud|.
+
+To run the demo, execute the following command:
+
+.. code-block:: bash
+
+      ./start.sh
+
+After running the previous command, you can complete the following workflow
+in about 8 minutes using the |ccloud| CLI:
+
+.. note::
+
+   This example uses real resources in |ccloud|, and it creates and deletes
+   topics, service accounts, API keys, and ACLs.
+
+-  `Create a new environment and specify it as the default`_
+-  `Create a new Kafka cluster and specify it as the default`_
+-  `Create a user key/secret pair and specify it as the default`_
+-  `Produce and consume with Confluent Cloud CLI`_
+-  `Create a service account key/secret pair`_
+-  `Run a Java producer: before and after ACLs`_
+-  `Run a Java producer: showcase a Prefix ACL`_
+-  `Run |kconnect| and kafka-connect-datagen connector with permissions`_
+-  `Run a Java consumer: showcase a Wildcard ACL`_
+-  `Delete the API key, service account, Kafka topics, Kafka cluster, environment,
+   and the log files`_
+
+
+Create a new environment and specify it as the default
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Run the following command to create a new |ccloud| environment
+   ``demo-script-env``:
+
+   .. code-block:: bash
+
+      ccloud environment create demo-script-env -o json
+
+   The output is in JSON format:
+
```

Comment:
I'm not too sure if this is all what we want to say for the JSON data here.  Should there be a step for the user view the JSON file and verify the data keys/values are set? @ybyzek. This also occurs in a few other spots in our page here. So what we decide here, I'll make the updates in the rest of the page accordingly

Reply:
@tikimims How about one step to take action (e.g. `run the command...`) and one step to verify output (e.g. `verify that the output resembles...`)?  Note, there is not JSON "file", it's just command *output* formatted as JSON>

Reply:
Cool beans. Updated